kroxylicious-api/src/main/java/io/kroxylicious/proxy/filter/FilterContext.java

```diff
@@ -23,11 +23,17 @@
  */
 public interface FilterContext {
     /**
-     * A description of this channel.
+     * A description of the downstream/client channel.
      * @return A description of this channel (typically used for logging).
      */
     String channelDescriptor();
 
+    /**
+     *
+     * @return the ID allocated to this client session.
+     */
```

Comment:
```suggestion
    /**
     * A unique id which uniquely identifies the connection with the client in both time and space.
     * In other words this will have a different value even if a client re-establishes a 
     * TCP connection from the same IP address and source port.
     * @return the ID allocated to this client session.
     */
```